sections/monte_carlo/quantum_monte_carlo/quantum_monte_carlo.tex

```diff
@@ -69,24 +61,246 @@ \subsubsection{Amplitude Estimation}\label{sec:MC_AE}
+    circuit = QuantumCircuit(qr_eval, qr_state, name=name)
+    iqft = QFT(num_evaluation_qubits, inverse=True, do_swaps=False).reverse_bits()
+    circuit.h(qr_eval)  # hadamards on evaluation qubits
+
+    for j in range(num_evaluation_qubits):  # controlled powers
+        circuit.append(unitary.power(2**j).control(), [j] + qr_state[:])
+    circuit.append(iqft.to_gate(), qr_eval[:])  # final QFT
+    return circuit
+\end{lstlisting}
+
+\subsubsection{Amplitude Amplification}\label{sec:amplitudeAmplification}
+
+
+As already mentioned amplitude amplification is the general case of grovers algorithmn. It amplifies the probability to measure a "good" part of a algorithm.\\
+This algorithm is in our case the circuit A. Suppose it initilizes n qubits, such that
+\begin{equation}\label{eq:amplitudeAmplA}
+  \ket{\phi} = A\ket{0}_{n} = \sqrt{1-a}\ket{\phi_0}_{n} +\sqrt{a}\ket{\phi_1}_{n},
+\end{equation}
+where $\ket{\phi_0}, \ket{\phi_1}$ are both normalized n-qubit states and orthorgonal to each other.
+For example could it be that the last qubit of $\ket{\phi_0} \text{ } \ket{0}$ and $\ket{\phi_1} \text{ } \ket{1}$ is.
+Let us denote $\ket{\phi_1}$ as the good-state, what means that the probability $\sqrt{a}$ should be increased.\\
+Furthermore $\ket{\phi_0}$ and $\ket{\phi_1}$ can be viewed as the horizontal and vertical axes of a two-dimensional Hilbert space. The angle between $\ket{U}$ and the horizontal axis is thus
+$\theta_a = \arcsin(\sqrt{a})$. Obviously we would like to rotate the initial state $\ket{\phi}$ towards the good-state, i.e. to the e.g. the vertical axis.\\
+The rotation is implemented as for Grover by defining a product of two reflections: a reflection trough the bad state and a reflection through $\ket{U}$.
+The first reflection is realized by a circuit $S_{\phi_0}$ which can distinguish the good from the bad state by putting a minus in front of $\ket{\phi_1}$ and leaving $\ket{\phi_0}$ alone.
+In case the last qubits are $\ket{1}, \ket{0}$ for $\ket{\phi_1}, \ket{\phi_0}$ respectively, then the reflection would be
+\begin{equation}\label{eq:PhiReflection}
+  S_{\psi_0} = 1-2\ket{\psi_0}\ket{0}\bra{\psi_0}\bra{0},
+\end{equation}
+and can in turn be realized by applying a Z-gate to the last qubit.\\
+The second reflection can be implemented as $AR_{0}A^{\dagger} = AR_0A$ with $R_0$
+\begin{equation}\label{eq:ZeroReflection}
+  R_0 = 2\ket{0^n}\bra{0^n} - \mathbb{1}.
+\end{equation}
+ $AR_0A$ puts a minus sign in front of states orthorgonal to $\ket{U}$. The product of both reflection is then (thanks to Grover) a rotation of angle
+ $2\theta$ in the two-dimensional picture introduced above. \\
+ The following amplitude amplification procedure increases the amplitude of the good state close to 1:
+\begin{enumerate}
+  \item{Setup the starting state $\ket{U} = A\ket{0^n}$}.
+  \item{Repeat the following $\mathcal{O}(1 \sqrt{a})$}.
+  \begin{itemize}
+    \item Reflect through bad state $\ket{\phi_0}$ ($R_G$).
```

Comment:
R_G is the same as S_{\phi_0} I think. One name should be used to avoid confusion